src/main/java/com/cloudbees/jenkins/plugins/amazonecs/ECSService.java

```diff
@@ -307,7 +307,7 @@ TaskDefinition registerTemplate(final String cloudName, final ECSTaskTemplate te
 
             if (template.isFargate()) {
                 request
-                        .withRequiresCompatibilities(template.getLaunchType())
+                        .withRequiresCompatibilities("FARGATE")
```

Comment:
Should we use a constant for "FARGATE" and "awsvpc"

Reply:
Not sure how to do that properly. (not from a development background). Do you have a sample somewhere in the code, so that I can refer.

Reply:
You can re-use the enum in com.amazonaws.services.ecs.model
Example, for "FARGATE" you can use the enum in com.amazonaws.services.ecs.model.LaunchType
And for "awsvpc", you can use the enum in  com.amazonaws.services.ecs.model.NetworkMode

Reply:
Fixed